### PR TITLE
Synchronise command adjustment

### PR DESCRIPTION
**Pull request by kfurgol**
**Description**

Changes proposed in this pull request:

- Changed --output flag to --dir flag to keep consistent with `init` command
- Name of the function is now passed as the only argument instead of being passed as a flag

**Related issue(s)**

### CODE REVIEW

File: cmd/kyma/sync/function/function.go [hunk 1]
```diff
@@ -30,20 +30,24 @@ func NewCmd(o *Options) *cobra.Command {
...
 Use the flags to specify the name of your Function, the Namespace, or the location for your project.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return c.Run()
+			return c.Run(args[0])
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return errors.New("missing name of the function")
+			}
+			return nil
```
Comment (clebs) [inline cmd/kyma/sync/function/function.go:39]:
```suggestion
		Args: cobra.ExactArgs(1)
```

Comment (clebs) [reply cmd/kyma/sync/function/function.go:39]: Only suggested if error message is not so relevant, otherwise please ignore.

**Review by clebs — APPROVED** [review]

File: cmd/kyma/sync/function/function.go (hunk 1 above)
Comment (kfurgol) [reply cmd/kyma/sync/function/function.go:39]: I didn't use `ExactArgs` on purpose to have the custom error msg, but I will change it if you insist on it

Comment (clebs) [reply cmd/kyma/sync/function/function.go:39]: As I said, only suggested if error message was is not relevant. If it is, leave it as it is.

**Review by kazydek — APPROVED** [review]

**Pull request merged**